### PR TITLE
Hypermarket data pipeline initial draft

### PR DESCRIPTION
**Pull request by chintalavr**
This PR includes following documentation

1. Microsoft Fabric workspace setup instructions
2. Commercial data pipeline
3. Operational data pipeline

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md [hunk 1]
```diff
@@ -205,6 +205,72 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
 
   ![Screenshot showing Agora resources in Azure portal](./img/rg_complete.png)
 
+### Setup Microsoft Fabric workspace
+
+Due to some limitations of automating Microsoft Fabric items in the Fabric workspace using managed identity, users deploying Contoso Hypermarket have to run a Microsoft Fabric workspace setup script manually using the end user credentials that are used to access Microsoft Fabric workspace.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md:210]:
```suggestion
Due to some limitations of automating Microsoft Fabric items in the Fabric workspace using a managed identity, users deploying Contoso Hypermarket have to run a setup script manually using the end user credentials that are used to access Microsoft Fabric workspace.
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md [hunk 2]
```diff
@@ -205,6 +205,72 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
...
+
+In order to create Microsoft Fabric workspace, tenant in which workspace is created must have one of the following settings enabled. Users can verify these settings in [Microsoft Fabric Admin Portal](https://app.powerbi.com/admin-portal/tenantSettings?experience=power-bi). If you do not have permissions to access Microsoft Fabric Admin Portal, please contact your Entra ID tenant to confirm these settings.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md:212]:
```suggestion
In order to create the Microsoft Fabric workspace, the tenant in which the workspace is created must have one of the following settings enabled. Users can verify these settings in [Microsoft Fabric Admin Portal](https://app.powerbi.com/admin-portal/tenantSettings?experience=power-bi). If you do not have permissions to access Microsoft Fabric Admin Portal, please contact your Entra ID tenant to confirm these settings.
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md [hunk 3]
```diff
@@ -205,6 +205,72 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
...
+
+  ![Screenshot showing PowerShell command-line tool launch](./img/ag-client-launch-powershell.png)
+
+- Change directory to _C:\Ag\Fabric_
+
+- Run the script file as shown below and follow the onscreen instructions to login to the Entra ID tenant.
+
+  ![Screenshot showing location of fabric setup PowerShell script](./img/fabric-run-script.png)
+
+- Users are prompted to complete authentication using code to log into the Entra ID tenant to create fabric workspace. Copy 1) device authentication URL and 2) Code from the command-line as shown below.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md:238]:
```suggestion
- Users are prompted to complete authentication using a device code to log into the Entra ID tenant to create fabric workspace. Copy 1) device authentication URL and 2) Code from the command-line as shown below.
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md [hunk 4]
```diff
@@ -205,6 +205,72 @@ If you already have [Microsoft Defender for Cloud](https://learn.microsoft.com/a
...
+
+  ![Screenshot showing device authentication URL and code](./img/fabric-device-authentication.png)
+
+- Open Edge browser and access the URL copied above and enter the code to complete authentication.
+
+  ![Screenshot showing device code prompt in the browser](./img/fabric-enter-device-code.png)
+
+- Select subscription when prompted to select as shown below and press enter
+
+  ![Screenshot showing subscription selection](./img/fabric-select-subsription.png)
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/deployment/_index.md:248]: I think the blurring is messed up, the image looks a little bit confusing

**Review by sebassem — CHANGES_REQUESTED** [review]
Minor modifications

**Comment by chintalavr** [comment]
I have updated as per the review comments

**Pull request merged**